### ISSUE TITLE
Is there a need to align the root joint before calculating the MPJPE?

### ISSUE THREAD

**Issue by axhiao** [Arthur151/SOTA-on-monocular-3D-pose-and-shape-estimation#1]
Hi, I don't know if there is a formal definition of MPJPE. But according to [here](https://nanonets.com/blog/human-pose-estimation-3d-guide/#appendix), they calculate the MPJPE after aligning the root joints.

**Comment by Arthur151** [comment]
Yes, normally, we calculate the MPJPE after aligning the root joints, which is pelvis in the most cases.
